R/Renderer.R

```diff
@@ -174,8 +174,17 @@ Renderer <- R6::R6Class( # nolint: object_name_linter.
     pictureBlock2md = function(block) {
       basename_pic <- basename(block$get_content())
       file.copy(block$get_content(), file.path(private$output_dir, basename_pic))
+      params <- c(
+        `out.width` = "'100%'",
+        `out.height` = "'100%'"
+      )
       title <- block$get_title()
-      sprintf("![%s](%s){width=%s, height=%s}", title, basename_pic, block$get_dim()[1], block$get_dim()[2]) # nolint
```

Comment:
we can solve this issue by removing the `,` between `width` and `height` as a comma is not required to separate them, see example:
`<img src="img_1.jpg" alt="description" width="500" height="600">`